calour/filtering.py

```diff
@@ -207,17 +213,18 @@ def filter_by_data(exp: Experiment, predicate, axis=0, negate=False, inplace=Fal
         The callable accepts a list of numeric and return a bool. Alternatively
         it also accepts the following strings:
 
-        * 'sum_abundance': calls ``_sum_abundance``,
-
-        * 'freq_ratio': calls ``_freq_ratio``,
-
-        * 'unique_cut': calls ``_unique_cut``,
-
-        * 'mean_abundance': calls ``_mean_abundance``,
-
-        * 'prevalence': calls ``_prevalence``
+        * 'sum_abundance': calls :func:`.sum_abundance`,
+        * 'mean_abundance': calls :func:`.mean_abundance`,
+        * 'prevalence': calls :func:`.prevalence`
+        * 'freq_ratio': calls :func:`.freq_ratio`,
+        * 'unique_cut': calls :func:`.unique_cut`
     axis : 0, 1, 's', or 'f', optional
-        Apply predicate on each row (samples) (0) or each column (features) (1)
+        Apply predicate on each row (ie samples) (0, 's') or each column (ie features) (1, 'f')
+    field : str or `None`, optional
+        The column in the sample_metadata (feature_metadata). If it is not `None`, the
+        features that has mean abundance lower than the cutoff in *ALL*
+        sample categories in the specified sample_metadata column will
+        be filtered away.
```

Comment:
In "features that has mean abundance lower than the cutoff...", is the field for mean_abundance or for any predicate?
Also should be "that have" rather than "that has"